lib/classes/Variables.js

```diff
@@ -344,11 +344,24 @@ class Variables {
     }, valueToPopulate);
   }
 
-  warnIfNotFound(variableString, valueToPopulate) {
+  deleteNullEnvironmentVariables() {
```

Comment:
Thanks for getting back @skewart 👍 

That sounds like a good idea. @HyperBrain what do you thing about it? In which plugin should be put it?